test/integration/selenium-helpers.js

```diff
@@ -293,18 +460,28 @@ class SeleniumHelper {
      * @returns {Promise<boolean>} True if the element's class attribute contains the given class, false otherwise.
      */
     async containsClass (element, cl) {
-        const classes = await element.getAttribute('class');
-        const classList = classes.split(' ');
-        return classList.includes(cl);
+        const outerError = new SeleniumHelperError('containsClass failed', [{element}, {cl}]);
+        try {
+            const classes = await element.getAttribute('class');
+            const classList = classes.split(' ');
+            return classList.includes(cl);
+        } catch (cause) {
+            throw await outerError.chain(cause, this.driver);
+        }
     }
 
     /**
      * @param {webdriver.WebElement} element Wait until this element is visible.
      * @param {webdriver.ThenableWebDriver} driver The webdriver instance.
      * @returns {Promise} A promise that resolves when the element is visible.
      */
-    async waitUntilVisible (element, driver) {
-        await driver.wait(until.elementIsVisible(element));
+    static async waitUntilVisible (element, driver) {
```

Comment:
This doesn't _have_ to be `static` but since it takes `driver` as a parameter, as opposed to using `this.driver` like all the other methods, I figured making it `static` would prevent confusion. Unfortunately callers treat it as an instance method, which leads to the weird-looking line in the constructor. It's basically equivalent to `bind`, though, so I'm not too upset.